### ISSUE TITLE
How to custom setting?

### ISSUE THREAD

**Issue by anhhtbk** [FolioReader/FolioReader-Android#96]
I want to change some setting, example: remove speaker, add new font, set default font, set default size..., How to do these?

**Comment by valinaga** [comment]
Hi everyone,

I spent like a whole day playing with Folio on Android. I'm far from an Android decent developer, but I'm quite familiar with java.
In order to customize, you need to import the folio reader and the webviewmaker into your project, and then tweak the source.
So, to remove speaker, you need to edit the folio_activity.xml and remove the last ineer LinearLayout (the one with btn_speaker). You may also comment or delete the method which throws an error upon compiling.
Adding new fonts is little more tricky, but playing with existing fonts and size, you need to edit the view_config.xml
Hope this helped.

**Comment by mahavir155** [comment]
@anhhtbk good idea. i think, it would be good if we provide some interfaces or params, so developer can change the settings without tweaking FR source..
